lib/l10n/intl_fr.arb

```diff
@@ -207,5 +207,15 @@
   "@sharing_after_uploaded_failure": {
     "type": "text",
     "placeholders": {}
+  },
+  "workgroup_nodes_surfing_root_back_title": "Back to Shared Space",
```

Comment:
Please lookup in our android project to find existed string localization first. If you can't find it then pls add missing translated string to this task: 
[https://ci.linagora.com/linagora/lgs/linshare/products/linshare-mobile-apps/issues/605](https://ci.linagora.com/linagora/lgs/linshare/products/linshare-mobile-apps/issues/605)